.gitignore

```diff
@@ -1,3 +1,3 @@
 target
 .env
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
We shouldn't add it to project gitignore. If we all add here all the artifacts of all OS, IDEs, etc, that would be hell
It should be in your global gitignore on your local machine